cfme/tests/automate/test_domain.py

```diff
@@ -79,16 +80,56 @@ def test_domain_lock_disabled(request, appliance):
         initialEstimate: 1/16h
         tags: automate
     """
-    domain = appliance.collections.domains.create(
-        name=fauxfactory.gen_alpha(),
-        description=fauxfactory.gen_alpha(),
-        enabled=False)
-    request.addfinalizer(domain.delete_if_exists)
-    domain.lock()
-    view = navigate_to(domain, 'Details')
+    # Disable automate domain
+    with update(klass.namespace.domain):
+        klass.namespace.domain.enabled = False
+
+    # Adding schema for executing automate method
+    klass.schema.add_fields({'name': 'execute', 'type': 'Method', 'data_type': 'String'})
+
+    # Adding automate method
+    method = klass.methods.create(
+        name=fauxfactory.gen_alphanumeric(),
+        display_name=fauxfactory.gen_alphanumeric(),
+        location='inline'
+    )
+
+    # Adding instance to call automate method
+    instance = klass.instances.create(
+        name=fauxfactory.gen_alphanumeric(),
+        display_name=fauxfactory.gen_alphanumeric(),
+        description=fauxfactory.gen_alphanumeric(),
+        fields={'execute': {'value': method.name}}
+    )
+
+    result = LogValidator(
+        "/var/www/miq/vmdb/log/automation.log",
+        matched_patterns=[r".*ERROR.*"],
```

Comment:
- According to me, there is no need to add that entire error line in ```matched patterns``` because logs generated in this session(like after executing automate method via simulation) have this only this error and if I got any other error then also it must fail. 
- So better to check no error only.
- Please let me know you thoughts @john-dupuy

Reply:
Yep that makes sense to me! Thanks for the clarification.